### ISSUE TITLE
"multiple" attribute not part of HTML Living Spec (as of 31 July 2016)

### ISSUE THREAD

**Issue by StephenBrown2** [LeaVerou/multirange#37]
It seems that as of some time between July 10, 2016, and July 31, 2016, the "multiple" attribute was removed from the input[type=range] spec, and is now listed as "does not apply".

Here is the latest web archive page with that text:
http://web.archive.org/web/20160710230742/https://html.spec.whatwg.org/multipage/forms.html#range-state-(type=range):attr-input-multiple-3

**Comment by ChALkeR** [comment]
Ref: https://github.com/w3c/html/issues/90

**Comment by LeaVerou** [comment]
Yup, that's true, which is why I haven't been very interested in maintaining this polyfill.

**Comment by AvnerCohen** [comment]
@LeaVerou Or anyone else, Any reference/background to why multiple was removed from the spec?
This really feels like a shortcoming of HTML.
To me, It makes little sense that "range" form field does not allow to select a "range" and only allows to select a specific value inside-a-range.

I can't find any discussion or future of this and given the popularity of non native range libs, I am quite sure this is a common need.

**Comment by basher** [comment]
I just stumbled upon this polyfill today, and was initially quite excited as it polyfills `multiple`. Or not. Hmmm.

I'm currently getting around the issue by using 2 separate `range` sliders side-by-side, but they don't look pretty.
